app/src/components/common/TmSessionImportName.vue

```diff
@@ -6,11 +6,7 @@
       </h2>
       <div class="session-main bottom-indent">
         <Steps :steps="[`Recover`, `Name`, `Password`]" active-step="Name" />
-        <TmFormGroup field-id="import-name" field-label="Your Address">
-          <HDPathOrCurve
```

Comment:
:(

Reply:
this component didn't make that much sense :shrug: